docs/documents.md

```diff
@@ -79,6 +79,15 @@ db.saveDoc('reports', {
 
 If the table already exists, you can still use `db.saveDoc`, but you can also invoke `saveDoc` on the table itself.
 
+`saveDocs` can be used for saving multiple documents.
```

Comment:
Can you add to this that the documents being saved have to be all new or all existing?

Reply:
Since the function also exists on `Writable` a note about it would also be good under the writable-level `saveDoc` section near the end of this file.